BUILDGUIDE.md

```diff
@@ -56,7 +56,7 @@ Once the environment is setup properly, execute the desired set of commands belo
 ```
 
 ```bash
-> msbuild /t:BuildAllOSes
+> msbuild /t:BuildNetCoreAllOS
 # Builds the driver for all Operating Systems.
```

Comment:
The comment and the target names don't agree about what they are doing.

Reply:
Updated the comment.